bedrock/mozorg/templates/mozorg/internet-health.html

```diff
@@ -194,7 +189,7 @@
     </section>
   </div>
 
-{% if l10n_has_tag('mozorg-health-report') and switch('internet-health-report-live') %}
```

Comment:
Yeah, we need to flip this switch today when internethealthreport.org goes live, then it'll be safe to remove. So this PR can't actually merge yet, I was hasty in removing the DNM flag.